app/lib/server/server.go

```diff
@@ -144,10 +146,17 @@ func InteractiveConfig(binaryName string) {
 
 	// create the keys
 	privStr, pubStr := createKeyPair()
+	ips, err := net.LookupIP(publicAddress.Host())
+	log.ErrFatal(err)
+	cert, key, err := network.NewCertKey(network.NewTLSCert(big.NewInt(1), "ch", "epfl", "dedis",
```

Comment:
Nodes will not only be ran by the org/domain dedis.epfl.ch ...  For tests it's OK to hardcode this but not for actual code.